src/vfocus.js

```diff
@@ -300,3 +300,13 @@ VFocus.prototype.children = function(){
 VFocus.prototype.addChild = function(child){
   this.vNode.children.push(child);
 };
+
+VFocus.prototype.indexOf = function(focus){
```

Comment:
Since focuses know their parent, how about we make this a bit "higher level"? We could instead ask the vFocus which index it has. Something like:
```
VFocus.prototype.indexOf = function() {
  return this.parent.vNode.children.indexOf(this.vNode);
}
```

So you'd simply call it like: `focus.indexOf()`.

That'd also allow us to reuse this method internally in VFocus, since we actually do this in 4 places in that file.